src/Core/Framework/Resources/config/packages/feature.yaml

```
@@ -66,9 +66,13 @@ shopware:
         major: true
         toggleable: true
         description: "This allows repeated calls to the /payment/finalize-transaction endpoint with the same token without throwing a 'token expired' exception."
-
       - name: METEOR_TEXT_EDITOR
         default: false
         major: false
         toggleable: true
-        description: "Enables the Meteor text editor in the administration."
\ No newline at end of file
+        description: "Enables the Meteor text editor in the administration."
+      - name: MULTI_CONTEXT_TOKENS
+        default: false
+        major: false
```

```suggestion
        major: true
```

if this should be enabled from 6.8 on